DTS-STN/Alpha-Site
# Move splash and index pages

# Description

[Move splash page to index and index to home](https://trello.com/c/UOQGlHam)

Splash page is moved to `/index` so it will be landing page. If a user has a language already set, it will redirect them to `/home` (formerly `/index`).

I did encounter what might be a bug with these redirects though, but I thought it out of scope of this PR. When a user visits a page (eg. `/experiments`) and they don't have a language set, they are redirected to the splash page to set the language. The bug is that the buttons to set the language redirect to `/home` rather than where they originally where trying to visit.

## Acceptance Criteria

`/splash` is gone and has been moved to `/`. The former index page is now at `/home`.

## Test Instructions

1. Navigate to `/`
2. If you have language in local storage already, you will be redirected to `/home`
3. If you haven't, notice it is now the splash page
4. The former `/` page is now at `/home`

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
